docs/data-ai/ai/create-dataset.md

```diff
@@ -14,244 +14,169 @@ aliases:
+To create a training set for object detection, add bounding boxes to teach your model to identify objects that you want to detect in future images.
 
-Repeat this with all images.
-{{< /expand >}}
+To label an object with a bounding box:
 
-## Organize data into a dataset
+1. Click on an image, then click the **Annotate** button in right side menu.
+1. Choose an existing label or create a new label.
+1. Holding the command key (on macOS), or the control key (on Linux and Windows), click and drag on the image to create the bounding box:
 
-To train a model, your images must be in a dataset.
+   {{<gif webm_src="/services/data/label-magnemite.webm" mp4_src="/services/data/label-magnemite.mp4" alt="Add a bounding box around the magnemite pokemon in an image">}}
 
-Use the interface on the **DATA** tab to add your labeled images to a dataset.
+{{< alert title="Tip" color="tip" >}}
 
-Also add any unlabelled images to your dataset.
-Unlabelled images must not comprise more than 20% of your dataset.
-If you have 25 images in your dataset, at least 20 of those must be labelled.
+Once created, you can move, resize, or delete the bounding box.
+{{< /alert >}}
 
-{{< expand "Want to add images to a dataset programmatically? Click here." >}}
+Repeat these steps for all images in the dataset.
 
-You can also add all images with a certain label to a dataset using the [`viam dataset data add` command](/dev/tools/cli/#dataset) or the [Data Client API](/dev/reference/apis/data-client/#addtagstobinarydatabyfilter):
+## Add tagged images to a dataset
 
 {{< tabs >}}
+{{% tab name="Viam app" %}}
+
+1. Open the **DATA** page of the Viam app.
+
+1. Navigate to the **ALL DATA** tab.
+
+1. Use the checkbox in the upper left of each image to select labeled images.
+
+1. Click the **Add to dataset** button, select a dataset, and click the **Add ... images** button to add the selected images to the dataset.
+
+{{% /tab %}}
 {{% tab name="CLI" %}}
 
-```sh {class="command-line" data-prompt="$"}
-viam dataset create --org-id=<org-id> --name=<name>
-viam dataset data add filter --dataset-id=<dataset-id> --tags=red_star,blue_square
-```
+Use the Viam CLI to filter images by label and add the filtered images to a dataset:
+
+1. First, run the following command to create a dataset, if you haven't already:
+
+   ```sh {class="command-line" data-prompt="$"}
+   viam dataset create --org-id=<org-id> --name=<dataset-name>
+   ```
+
+   This command will output a dataset ID.
+   Pass this dataset ID in the next step to add images to this dataset.
+
+1. Run the following [command](/dev/tools/cli/#dataset) to add all images labeled with a subset of tags to the dataset, replacing the `<dataset-id>` placeholder with the dataset ID output by the command in the previous step:
+
+   ```sh {class="command-line" data-prompt="$"}
+   viam dataset data add filter --dataset-id=<dataset-id> --tags=red_star,blue_square
+   ```
 
 {{% /tab %}}
 {{< tab name="Data Client API" >}}
```

Comment:
This tab is currently broken. Try to change the `<` and `>` to `%` and see if that fixes it.

Reply:
Fixed!